src/pybind/mgr/dashboard/frontend/src/app/shared/components/deletion-modal/deletion-modal.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, OnInit, Output, TemplateRef } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { BsModalRef, BsModalService } from 'ngx-bootstrap';
+
+@Component({
+  selector: 'cd-deletion-modal',
+  templateUrl: './deletion-modal.component.html',
+  styleUrls: ['./deletion-modal.component.scss']
+})
+export class DeletionModalComponent implements OnInit {
+  @Input() metaType: string;
+  @Input() pattern: string;
+  @Input() btnClasses = 'btn btn-primary';
```

Comment:
Should be 'btn btn-sm btn-primary' (dashboard buttons are small).

Reply:
This is correct, but the button still does not fit in a btn-group. I think we need to duplicate the btn-group CSS for cd-deletion-modal.
![bildschirmfoto vom 2018-03-26 09-44-41](https://user-images.githubusercontent.com/1897962/37893189-94ca5f60-30da-11e8-91cf-de6ad87539a3.png)

Reply:
fixed